DQM/DTMonitorModule/python/dtTriggerEfficiencyTask_cfi.py

```diff
@@ -12,12 +12,12 @@
     inputTagDDU = cms.untracked.InputTag('muonDTDigis'),
     inputTagSEG = cms.untracked.InputTag('dt4DSegments'),
     inputTagGMT = cms.untracked.InputTag('gtDigis'),
-    processDDU = cms.untracked.bool(True),  # if true enables DDU data analysis
+    processDDU = cms.untracked.bool(False),  # Not needed any longer. Switches below for Eras do not work... 
     processTM = cms.untracked.bool(True), # if true enables TM data analysis
     minBXDDU = cms.untracked.int32(7),  # min BX for DDU eff computation
     maxBXDDU = cms.untracked.int32(15), # max BX for DDU eff computation
 
-    checkRPCtriggers = cms.untracked.bool(True),
+    checkRPCtriggers = cms.untracked.bool(False), #  Not needed any longer. Switches below for Eras do not work...
```

Comment:
hi @davidlange6 

I had to manually disable the check on RPC triggers since the lines below 30-37 to switch depending on the input were not working: i.e. if the input file is a 2016 25ns run, they are ignored. Is that expected or the Eras switch only works in real workflow productions?
Thanks